src/promql/src/planner.rs

```diff
@@ -1847,11 +1853,13 @@ impl PromPlanner {
             .zip(self.ctx.field_columns.iter())
             .map(|(expr, name)| Ok(DfExpr::Alias(Alias::new(expr, name.to_string()))));
 
-        // chain non-value columns (unchanged) and value columns (applied computation then alias)
+        // chain non-field columns (unchanged) and field columns (applied computation then alias)
         let project_fields = non_field_columns_iter
             .chain(field_columns_iter)
             .collect::<Result<Vec<_>>>()?;
 
+        common_telemetry::info!("[DEBUG] project fields: {:?}", project_fields);
```

Comment:
Remove the log or change it to debug level

Reply:
ah, I forget this